core/chains/evm/config/toml/defaults/Arbitrum_Sepolia.toml

```diff
@@ -1,5 +1,6 @@
 ChainID = '421614'
 ChainType = 'arbitrum'
+FinalityDepth = 10
```

Comment:
If a lot of the chains use 10 as default, I think we should modify the fallback value to 10 & we can remove it from here, @prashantkumar1982 @dimriou any inputs?

Reply:
Currently default is set to 50 for Eth mainnet.
I'm ok with anything.

Reply:
Removed.